about/introduction.rst

```diff
@@ -6,7 +6,7 @@ Introduction
 ::
 
     func _ready():
-        get_node("Label").set_text("Hello world!")
+        $Label.set_text("Hello world!")
```

Comment:
I think it would be better to also use members, so `$Label.text = "Hello World!"`.

Reply:
lol obviously...